### PR TITLE
fix(VWindow): fix wrong reverse animation logic for only 2 items

### LINKED ISSUE #12672: [Bug Report] Two tabs transition reversed
### Environment
**Vuetify Version:** 2.3.19
**Last working version:** 2.3.18
**Vue Version:** 2.6.12
**Browsers:** Firefox 83.0
**OS:** Ubuntu undefined

### Steps to reproduce
Make tabs with only 2 of them. (3 or more is OK)

### Expected Behavior
Tabs item should go right to left. And then switching left to right.

### Actual Behavior
Tab items go left to right. And then switching right to left.

### Reproduction Link
<a href="https://codepen.io/bpuig/pen/ZEpzorg" target="_blank">https://codepen.io/bpuig/pen/ZEpzorg</a>

### PR DESCRIPTION
**Pull request by yuwu9145**
fixes #12672

## Description

Issue  #12672 is caused by a recent change from [PR 12357](https://github.com/vuetifyjs/vuetify/pull/12357). Its change results in wrong animation reverse logic when items length is less than 3. 

This pull request is making it fully support all items length in terms of animation reverse logic on top of the code from [PR 12357](https://github.com/vuetifyjs/vuetify/pull/12357). 

@nekosaur please help to check it out to see this change would raise any concerns from you and please also feel free to leave any feedback :). 

## Motivation and Context

closes #12672

## How Has This Been Tested?

visually

## Markup:

<details>

```vue
<template>
  <v-container>
    <v-card>
      <v-toolbar flat>
        <template v-slot:extension>
          <v-tabs
            v-model="tabs"
            fixed-tabs
          >
            <v-tabs-slider></v-tabs-slider>
            <v-tab
              href="#mobile-tabs-5-1"
              class="primary--text"
            >
              <v-icon>mdi-phone</v-icon>
            </v-tab>

            <v-tab
              href="#mobile-tabs-5-2"
              class="primary--text"
            >
              <v-icon>mdi-heart</v-icon>
            </v-tab>
          </v-tabs>
        </template>
      </v-toolbar>

      <v-tabs-items v-model="tabs">
        <v-tab-item
          v-for="i in 2"
          :key="i"
          :value="'mobile-tabs-5-' + i"
        >
          <v-card flat>
            <v-card-text v-text="text"></v-card-text>
          </v-card>
        </v-tab-item>
      </v-tabs-items>
    </v-card>
  </v-container>
</template>

<script>
  export default {
    data () {
      return {
        tabs: null,
        text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
      }
    },
    methods: {
    }
  }
</script>
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VWindow/VWindow.ts [hunk 1]
```diff
@@ -222,12 +222,17 @@ export default BaseItemGroup.extend({
       this.internalValue = this.getValue(item, lastIndex)
     },
     updateReverse (val: number, oldVal: number) {
-      const lastIndex = this.items.length - 1
-
-      if (val === lastIndex && oldVal === 0) {
-        return true
-      } else if (val === 0 && oldVal === lastIndex) {
-        return false
+      const itemsLength = this.items.length
```
Comment (yuwu9145) [inline packages/vuetify/src/components/VWindow/VWindow.ts:225]: @nekosaur , please help to check if this change is compatible with fixing issue #11280 from this [PR](https://github.com/vuetifyjs/vuetify/pull/12357).

Thank you in advance :)

File: packages/vuetify/src/components/VWindow/VWindow.ts [hunk 2]
```diff
@@ -222,7 +222,10 @@ export default BaseItemGroup.extend({
       this.internalValue = this.getValue(item, lastIndex)
     },
     updateReverse (val: number, oldVal: number) {
-      const lastIndex = this.items.length - 1
+      const itemsLength = this.items.length
+      const lastIndex = itemsLength - 1
+
+      if (itemsLength <= 2) return val < oldVal
```
Comment (yuwu9145) [inline packages/vuetify/src/components/VWindow/VWindow.ts:228]: @nekosaur , please help to check if this change is compatible with fixing issue #11280 from this PR.

Thank you in advance :)

**Comment by yuwu9145** [comment]
@MajesticPotatoe @nekosaur , I checked out myself, this PR respects the fix for this [issue](https://github.com/vuetifyjs/vuetify/issues/11280) from [PR 12357](https://github.com/vuetifyjs/vuetify/pull/12357)

**Review by MajesticPotatoe — APPROVED** [review]
👍 
Thank you for your contribution and interest in improving Vuetify! Make sure to join us in the [Discord community](https://community.vuetifyjs.com).

**Pull request merged**